pkg/exchange/bitget/exchange.go

```diff
@@ -29,12 +35,15 @@ var (
 	queryTickerRateLimiter = rate.NewLimiter(rate.Every(time.Second/10), 5)
 	// queryTickersRateLimiter has its own rate limit. https://bitgetlimited.github.io/apidoc/en/spot/#get-all-tickers
 	queryTickersRateLimiter = rate.NewLimiter(rate.Every(time.Second/10), 5)
+	// queryOpenOrdersRateLimiter has its own rate limit. https://www.bitget.com/zh-CN/api-doc/spot/trade/Get-Unfilled-Orders
+	queryOpenOrdersRateLimiter = rate.NewLimiter(rate.Every(time.Second/10), 5)
 )
 
 type Exchange struct {
 	key, secret, passphrase string
 
-	client *bitgetapi.RestClient
+	client   *bitgetapi.RestClient
```

Comment:
is this v3? call it v3client?

Reply:
No, i assume the client is v1, so i don't want to rename it

Reply:
I think it will be deprecated soon...